.github/workflows/dusk_ci.yml

```diff
@@ -21,18 +21,17 @@ jobs:
           command: dusk-analyzer
 
   test_nightly:
-    name: Nightly tests Debug Mode
+    name: Nightly tests
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: nightly
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --all-features
+          override: true
```

Comment:
This should be removed (we have the toolchain file now)

Reply:
what exactly should be removed?

Reply:
you don't need to `override` anymore neither to specify a `toolchain`, since we have the file.
You basically have only `profile: minimal` here.